### PR TITLE
feat(wizard-header): add component

### PR DESCRIPTION
**Pull request by pedrorebelocosta**
Default:
image

### CODE REVIEW

**Review by ricardogoncalves89 — APPROVED** [review]

File: packages/ui-components/src/types.ts [hunk 1]
```diff
@@ -35,3 +35,4 @@ export * from './components/toggle-tip/toggle-tip.types';
 export * from './components/tooltip/tooltip.types';
 export * from './components/tag-status/tag-status.types';
 export * from './components/tag-alarm/tag-alarm.types';
+export * from './components/wizard-header/wizard-header.types';
```
Comment (djcouto) [inline packages/ui-components/src/types.ts:38]: Don't forget to add the angular component.

Comment (pedrorebelocosta) [reply packages/ui-components/src/types.ts:38]: Adding a wizard here is indeed the plan :P I have to make some changes to this component due to the redesign.
Thanks for pointing out the lack of export on the angular lib, totally missed that.

**Comment by ricardogoncalves89** [comment]
>

**Comment by ricardogoncalves89** [comment]
> Are we planning to add a wizard to the UI components? If not, maybe we should generalize this component name 😄
> 
> Also, left you a comment.

Yes @djcouto, I will create a KvWizard here.

**Review by djcouto — APPROVED** [review]

**Pull request merged**